src/index.css

```diff
@@ -0,0 +1,8 @@
+body {
+    background-image: linear-gradient(#8968ac, #fcc5f9);
+}
+
+.contentWrapper {
+    margin: 0 5rem;
```

Comment:
alphabetical order for these properties. we may be able to update your linter to do this automatically